src/odin/narrativebuilder.cc

```diff
@@ -79,7 +79,7 @@ void NarrativeBuilder::Build(std::list<Maneuver>& maneuvers) {
 
         // Set verbal succinct transition instruction
         maneuver.set_verbal_succinct_transition_instruction(
```

Comment:
If the <DESTINATION> string is large then we will want to set the succinct to either phrase `0` or `2`
```
  // "0": "You have arrived at your destination."
  // "2": "Your destination is on the <RELATIVE_DIRECTION>."
```
Otherwise, we do not want to set succinct instruction for the destination maneuver